### ISSUE TITLE
[Bug] Unsynced Modifier Not Working (for real this time)

### ISSUE THREAD

**Issue by Katsuvera** [ffxivcode/AutoDuty#379]
### What happened?

TL;DR: It is doing Unsynced actions while Synced (in Duty Support, at the very least).

To clarify, Synced actions seem to work just fine (did one Unsynced run to test and it skipped all the Synced actions just fine).

I tested across multiple runs, made sure the in-game "Unrestricted Party" setting was off, and made sure the "Unsynced" button in the plugin was Off in the "Regular" Duty Mode place (just in case). It's still doing Unsynced actions.

### Version Number

150

### Steps to reproduce the error

Try the following Path in Sirensong Sea with Duty Support. If you run to the right at the start, it's doing the first UnsyncedMoveTo step.

{
  "actions": [
    "<-- Trash Packs #1 & #2 -->",
    "UnsyncedMoveTo|-8.68, 21.10, -470.09|",
    "StopForCombat|6.98, 16.35, -456.29|False",
    "StopForCombat|2.27, 16.35, -421.01|True",
    "Wait|0, 0, 0|10000",
    "StopForCombat|0.02, 23.85, -394.89|False",
    "Wait|0, 0, 0|8000",
    "MoveTo|-1.67, 26.18, -379.35|"
      ],
  "meta": {
    "createdAt": 150,
    "changelog": [],
    "notes": []
  }
}

### Other relevant plugins installed

It doesn't show the full thing but here's an image of me clearly being in a party and it's on (and doing) an UnsyncedMoveTo step.
![image](https://cdn.discordapp.com/attachments/1070794391234551808/1280806564617256990/image.png?ex=66d96b91&is=66d81a11&hm=d9b1d235c770324fa7c49be0a74c7aefc8fa42b6c67c0c1d852c70016938697a&)

**Comment by Katsuvera** [comment]
Alright, I just ran another run Unsynced and it is _not_ working properly. aka. It _is_ doing the Synced steps. aka. it's doing both the Synced and Unsynced steps. So neither is working properly.

But the thing that doesn't make sense is that I _know_ I was doing Unsynced runs and it was working perfectly fine earlier. I recorded it working properly so I know I'm not crazy. lol And yes, it's all on v150, I don't mind that long ago when I said "earlier".

I guess I'll, uh, re-start my game completely and see if Unsynced runs work again... Honestly, I have no idea what's happening now. I'm not even sure which part of the /xllog would be relevant, either...

**Comment by Katsuvera** [comment]
I re-started the game and it's back to Unsynced runs working fine (skipping Synced steps), but Synced still not working (doing Unsynced steps).

**Comment by Katsuvera** [comment]
Tried checking the /xllog when it starts an Unsynced action while Synced but unfortunately it only seems to say

21:49:21.693 | DBG | [BossModReborn] Condition change: InCombat=False
21:49:21.912 | DBG | [AutoDuty] Stage=Reading Path
21:49:21.927 | INF | [vnavmesh] Queueing move-to <-43.140, 5.970, -43.040>
21:49:21.927 | DBG | [AutoDuty] Stage=Moving
21:49:21.941 | DBG | [vnavmesh] [pathfind] poly 1000000900075 -> 100000080007D
21:49:21.941 | DBG | [vnavmesh] Pathfind took 0.000s: 1000000900075, 1000000900076, 100000090006E, 1000000900086, 1000000900084, 1000000900089, 1000000900088, 1000000800080, 100000080007C, 1000000800077, 100000080007D
21:49:21.958 | INF | [vnavmesh] Pathfinding complete
